app/models/notifications/EntryStatus.scala

```diff
@@ -16,10 +16,10 @@
 
 package models.notifications
 
-import models.notifications.queries.RoeCode
+import models.viewmodels.decoder.ROECode
 import play.api.libs.json.Json
 
-case class EntryStatus(ics: Option[String] = None, roe: Option[RoeCode] = None, soe: Option[String] = None)
+case class EntryStatus(ics: Option[String] = None, roe: Option[ROECode] = None, soe: Option[String] = None)
```

Comment:
Nit: We've had a discussion some time ago not to capitalise codes. So better stay with `RoeCode`

Reply:
ROECode is the class which already existed. RoeCode is the class that I created and removed in second comment. I'll rename this class and probably rest of the codes classes, but after rebasing to the Steve's branch.

If you look at the all changes you'll be able to see that I changed String to ROECode